app/controllers/admin/stats_controller.rb

```diff
@@ -0,0 +1,49 @@
+# -*- encoding : utf-8 -*-
+
+class Admin::StatsController < ApplicationController
+  def index
```

Comment:
This method is really long. Can you break it up into a few smaller methods? Perhaps: `recent_users()`, `active_users()` and `files_count()`